application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -44,12 +43,22 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
         if (tagSystem.handleIsUnknownTag(id, event)) {
             return;
         }
-
-        event.replyEmbeds(tagEmbed(id, event.getUser(), event.getCommandString())).queue();
+/*
+event.replyEmbeds(tagEmbed(id, event.getUser(), event.getCommandString())).queue();
     }
 
     private MessageEmbed tagEmbed(String id, User user, String commandString) {
         return MessageUtils.generateEmbed(null, tagSystem.getTag(id).orElseThrow(), user,
                 TagSystem.AMBIENT_COLOR, user.getName() + " " + commandString);
     }
+ */
+
+        event
+            .replyEmbeds(new EmbedBuilder().setDescription(tagSystem.getTag(id).orElseThrow())
+                .setFooter(event.getUser().getName() + " - used /tag foo")
```

Comment:
You need to use the commandString and not `- used /tag foo`. Have you even tested this?^^

Also, we wanted to use the bullet point and not `-`

Reply:
Yes I have, and pretty sure I have added a ss in the group how it looks like